src/service/routes/compatibility.js

```diff
@@ -11,9 +11,5 @@ router.get('/ui/wwe/profile.js', (req, res) => res.status(200).json({}))
 router.get('/auth/profile.js', (req,res) => res.status(200).json({}))
 router.get('/auth/telemetry.js', (req,res) => res.status(200).json({}))
 router.get('/auth/host-mapping', (req,res) => res.status(200).json({}))
-// to avoid an error in WWE
```

Comment:
Yes the code needed to be removed as it did not come into the voice route for send-user-event otherwise. Moreover, a default response is always sent for send-user-event requests in the in new code I added in voice route so removing this should not be an issue. I'll confirm with person who added this.